.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.3.8
```

Comment:
This project already has a `.ruby-version` (which was also upgraded to 2.5.9 since you opened this PR).

It might be easiest to just remove this file again, and follow the steps in the [asdf-ruby readme](https://github.com/asdf-vm/asdf-ruby#migrating-from-another-ruby-version-manager) to make it recognize `.ruby-version` files?